cmd/mimir-whisper-converter/Dockerfile

```diff
@@ -17,6 +17,15 @@ RUN curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o "awscliv2
   && unzip awscliv2.zip \
   && ./aws/install
 
+RUN apt install -y nano less jq
+
+RUN wget https://github.com/thanos-io/thanos/releases/download/v0.32.5/thanos-0.32.5.linux-amd64.tar.gz \
```

Comment:
Maybe you'd like to do this step in the previous build step and just copy the binary over

Reply:
sorry, I don't quite understand. Which build step? I was mimicking the curl install process

Reply:
![image](https://github.com/grafana/mimir-proxies/assets/2439858/b9848b2c-6826-480b-8268-9f162e1ee8ce)

On line 32 you can see how a binary was copied already 
`COPY --from=build /whisper/build /app/opt/whisper`

Just pointed this out as a nit because the resulting image is cleaner, leaner and more consistent with what was already happening 👍